### PR TITLE
main/qpdf: upgrade to 7.1.1, fix license

### PR DESCRIPTION
**Pull request by dai9ah**

### CODE REVIEW

File: main/qpdf/APKBUILD [hunk 1]
```diff
@@ -1,19 +1,29 @@
...
 pkgrel=0
 pkgdesc="Command-line tools and library for transforming PDF files"
-url="http://qpdf.sourceforge.net/"
+url="http://qpdf.sourceforge.net"
 arch="all"
-license="Artistic-2.0"
+license="Apache-2.0"
 depends=""
 # coreutils needed for tests
 makedepends="zlib-dev libjpeg-turbo-dev bash coreutils perl"
```
Comment (clandmeter) [inline main/qpdf/APKBUILD]: Can you move the tools needed for tests(check) to checkdepends?

Comment (dai9ah) [reply main/qpdf/APKBUILD]: Yes sir.
Moved coreutils to checkdepends.

File: main/qpdf/APKBUILD [hunk 2]
```diff
@@ -1,19 +1,33 @@
...
+subpackages="$pkgname-dev
+	$pkgname-doc
+	$pkgname-libs
+	$pkgname-fix-qdf:fix_qdf:noarch
+	"
+source="http://downloads.sourceforge.net/sourceforge/$pkgname/$pkgname-$pkgver.tar.gz"
+builddir="$srcdir/$pkgname-$pkgver"
 
-builddir="$srcdir"/qpdf-$pkgver
+# secfixes:
```
Comment (nmeum) [inline main/qpdf/APKBUILD]: Could you add the secfixes comment in a seperate commit with the commit message `main/qpdf: add secfixes comment`?

Comment (dai9ah) [reply main/qpdf/APKBUILD]: Added secfixes to a separate commit.

Comment (dai9ah) [reply main/qpdf/APKBUILD]: Yes. Added comments to a separate commit.

File: main/qpdf/APKBUILD [hunk 3]
```diff
@@ -1,19 +1,22 @@
...
 pkgdesc="Command-line tools and library for transforming PDF files"
-url="http://qpdf.sourceforge.net/"
+url="http://qpdf.sourceforge.net"
 arch="all"
-license="Artistic-2.0"
+license="Apache-2.0"
 depends=""
-# coreutils needed for tests
-makedepends="zlib-dev libjpeg-turbo-dev bash coreutils perl"
+makedepends="zlib-dev libjpeg-turbo-dev bash perl"
```
[the remark this answers is not in the document]
Comment (dai9ah) [reply main/qpdf/APKBUILD]: Perl is (optionally) required for both fix-qdf and the testsuites. If we omit perl, the fix-qdf program will no longer work.

[the remark this answers is not in the document]
Comment (clandmeter) [reply main/qpdf/APKBUILD]: fix-pdf is a subpkg and has perl included as dependency.

[the remark this answers is not in the document]
Comment (dai9ah) [reply main/qpdf/APKBUILD]: Indeed you are correct!
Moved perl to checkdepends.

**Pull request closed without merging**